test/models/work_test.rb

```diff
@@ -0,0 +1,187 @@
+require "test_helper"
+
+describe Work do
+  before do
+    @work = Work.new(title: "A Beautiful Mind", creator: "Akiva Goldsman", publication_year: "2001" , description: "American biographical drama film based on the life of the American mathematician John Nash.")
+    @lak = users(:lak)
+    @cathy = users(:cathy)
+  end
+
+  it "work can be instantiated" do
+    @work.save
+    expect(@work.valid?).must_equal true
+  end
+
+  it "work will have the required fields" do
+    work = works(:work1)
+
+    [:category, :title, :creator, :publication_year, :description].each do |field|
+      expect(work).must_respond_to field
+    end
+  end
+
+  describe "relations" do
+    it "work can have many votes" do
+      first_work = works(:work1)
+      expect {
+        Vote.create!(work_id: first_work.id, user_id: @lak.id)
+        Vote.create!(work_id: first_work.id, user_id: @cathy.id)
+      }.must_differ "first_work.votes.count", 2
+    end
+
+    it "work can have many users through votes" do
+      first_work = works(:work1)
+      Vote.create!(work_id: first_work.id, user_id: @lak.id)
+      Vote.create!(work_id: first_work.id, user_id: @cathy.id)
+
+      first_work.votes.each do |vote|
+        user = User.find_by(id: vote.user_id)
+        expect(user).must_be_instance_of User
+      end
```

Comment:
As has many through relationship should allow you to just call `users` directly on the vote:

```suggestion
      first_work.users.each do |user|
        expect(user).must_be_instance_of User
      end
```